### PR TITLE
feat(connect-explorer): update favicon image + trezor picture in header

### LINKED ISSUE #9743: Old Trezor logo used as connect favicon
Navigate to https://connect.trezor.io/9/#/ and check favicon

image

### PR DESCRIPTION
**Pull request by Hannsek**

### CODE REVIEW

**Comment by Hannsek** [comment]
Not sure about `width="100%" height="100%" preserveAspectRatio="xMinYMin meet"` parameters in SVG component.

Also the formatting is kinda strange, but I ran `yarn format` so it should be ok.

@mroz22 can you tell me why the test are failing? is it because the new `class="trezor-logo-trezor", class="trezor-logo-text"` ?

File: packages/connect-explorer/src/components/Header.tsx [hunk 1]
```diff
@@ -60,23 +60,15 @@ const Header = () => (
         <HeaderPrimary>
             <HeaderItemsPrimary>
                 <a href="#/">
-                    <Svg
```
Comment (tomasklim) [inline packages/connect-explorer/src/components/Header.tsx:63]: can we use .svg file instead of this?

Comment (mroz22) [reply packages/connect-explorer/src/components/Header.tsx:63]: if you show us how to.. 🤷

File: packages/connect-explorer/src/components/Header.tsx [hunk 2]
```diff
@@ -60,23 +60,15 @@ const Header = () => (
...
-                            <path d="M1869.7 720.3c0-104.6 81.1-176.4 186.5-176.4 105 0 186.5 71.4 186.5 176.4 0 104.6-81.1 176-186.5 176s-186.5-71.4-186.5-176zm268 0c0-47.5-32.3-85.3-81.9-85.3-49.6 0-81.9 37.8-81.9 85.3s32.3 85.3 81.9 85.3c50 0 81.9-37.8 81.9-85.3z" />
-                            <path d="M2473.6 890.4l-47-107.1h-37.4v107.1h-103.3V549.8h181.8c79.8 0 122.6 52.9 122.6 116.7 0 58.8-34 89.9-61.3 103.3l61.7 120.5h-117.1zm12.6-224.3c0-18.5-16.4-26.5-33.6-26.5h-63v53.8h63c17.3-.4 33.6-8.4 33.6-27.3z" />
-                        </g>
+                    <Svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 161.768 40.771">
+                        <path
+                            class="trezor-logo-trezor"
+                            d="M24.306 9.461C24.306 4.29 19.761 0 14.228 0 8.694 0 4.148 4.292 4.148 9.46v3.025H0v21.75l14.225 6.536 14.233-6.534V12.581H24.31l-.003-3.121Zm-15.02 0c0-2.438 2.175-4.389 4.942-4.389 2.767 0 4.94 1.951 4.94 4.389v3.024H9.287V9.461Zm13.44 21.264-8.502 3.904-8.499-3.901V17.655h17v13.07z"
+                        ></path>
+                        <path
+                            class="trezor-logo-text"
```
Comment (mroz22) [inline packages/connect-explorer/src/components/Header.tsx]: class should not be here unless it has some usecase

File: packages/connect-explorer/src/components/Header.tsx (hunk 1 above)
Comment (tomasklim) [reply packages/connect-explorer/src/components/Header.tsx:63]: add to suite-data, copy and import? but let's skip it :D

File: packages/connect-explorer/src/components/Header.tsx (hunk 2 above)
Comment (Hannsek) [reply packages/connect-explorer/src/components/Header.tsx]: fixed in cf9e89e8625c8df351dc80ca6e8f219837eb74a3

**Comment by Hannsek** [comment]
Also not sure about the `<g transform="translate(91.363 -287.434) scale(.95575)">` which was there previously. 🤔

**Comment by Hannsek** [comment]
The type-check_and_lint says that there are two self-closing components
```
/home/runner/work/trezor-suite/trezor-suite/packages/connect-explorer/src/components/Header.tsx
  64:25  error  Empty components are self-closing  react/self-closing-comp
  65:[25](https://github.com/trezor/trezor-suite/actions/runs/6849547647/job/18621962413#step:17:26)  
  error  Empty components are self-closing  react/self-closing-comp
```

however, they are not. They are properly closed: `<path d="M24.306 9.461C24.306 … … … "></path>`.
Is there some error in the CI? Or do I have there some mistake that I don't see?🙏🏻
@tomasklim @mroz22

**Comment by mroz22** [comment]
```
<something something="something'></something>
```
is selfclosing

**Comment by Hannsek** [comment]
I think that <something something="something" /> is self-closing component. I.e. it ends with "/".

**Comment by mroz22** [comment]
yep, I am not very clear. it wants you to change 
```
<path d="M24.306 9.461C24.306 … … … "></path>.
```
 to 
```
<path d="M24.306 9.461C24.306 … … … "/>
```
I'd say

**Review by mroz22 — APPROVED** [review]
ack, did not test

**Review by karliatto — APPROVED** [review]

**Pull request merged**